projects/envoy/build.sh

```diff
@@ -22,6 +22,8 @@ export CXXFLAGS="$CXXFLAGS -fno-sanitize=vptr"
 declare -r FUZZER_TARGETS_CC=$(find . -name *_fuzz_test.cc)
 declare -r FUZZER_TARGETS="$(for t in ${FUZZER_TARGETS_CC}; do echo "${t:2:-3}"; done)"
 
+echo FUZZER_TARGETS_CC
```

Comment:
I don't think this does anything useful; it should be `echo "${FUZZER_TARGETS_CC}"` if you want to keep it around.